src/app/lib/logger.node.js

```diff
@@ -17,29 +28,31 @@ const createLogDirectory = (dirName = 'log') => {
 
 const logLocation = path.join(LOG_DIR, LOG_FILE);
 
-// prettier-ignore
-const fileTransport = new (transports.File)({
-  filename: logLocation,
-  handleExceptions: true,
-  humanReadableUnhandledException: true,
-  json: true,
-  level: LOG_LEVEL,
-  maxFiles: 5,
-  maxsize: 104857600, // 100MB
-  tailable: true
-});
+const logFormat = printf(({ timestamp, level, label, message, metadata }) => {
```

Comment:
Can I make a couple of suggestions - just to make it similar to how it was previously:
label -> filename
message -> event (just so we don't have to update all our Sumo queries)

Reply:
I can change these values however it will not make a difference when it comes to the logs in Sumo.

Winston records the first parameter of the logs as `message` and the second as metadata. We got around this before as we didn't use the correct values/functions from winston (as far as I can tell).

So I think we are going to have to update our sumo queries regardless. At least we can start removing the regex from the queries now though.

This is an example of an object that should be sent to Sumo

```
{
  "message": "include_rendered",
  "level": "info",
  "timestamp": "2020-07-08 20:50:40.913",
  "label": "Include/index.jsx",
  "metadata": {
    "includeUrl": "/idt2/cb1a5166-cfbb-4520-bdac-6159299acff6",
    "type": "idt2"
  }
}
```

Reply:
Gotcha. I've already made a start at simplifying the queries in Sumo (removing the JSON parsing & just accessing the fields directly), and it's not difficult to update. Thanks.